.github/workflows/maven.yml

```diff
@@ -11,6 +11,12 @@ jobs:
         os: [ubuntu-latest, windows-latest, macos-latest]
 
     steps:
+      - name: Checkout powsybl-core sources
```

Comment:
I imagine that you have to clean that now. `Maven.yml` must be the one from master branch.